Makefile

```diff
@@ -0,0 +1,20 @@
+CPP     = g++
+CPPOPTS = -g -O3 -Wall -std=c++11 -Iinclude/ -Ideps/ -fprofile-arcs -ftest-coverage
```

Comment:
You may want debug with and without coverage since you don't want local dev runs to pollute your coverage measurements.